packages/ember-data/lib/adapters/build_url_mixin.js

```diff
@@ -0,0 +1,111 @@
+var get = Ember.get;
+
+export default Ember.Mixin.create({
```

Comment:
This will need a doc block at the top with an `@class BuildURLMixin` annotation for the methods below to get correctly attributed to the correct class.